README.md

```diff
@@ -1,78 +1,120 @@
-# Detection Metrics
-
-[![Publish Docker image](https://github.com/JdeRobot/DetectionStudio/actions/workflows/main.yml/badge.svg)](https://github.com/JdeRobot/DetectionStudio/actions/workflows/main.yml)
-
-#### More info and documentation [here](https://jderobot.github.io/DetectionMetrics/).
-
-Detection Metrics is a set of tools to evaluate object detection neural networks models over the common object detection datasets.
-The tools can be accessed using the GUI or the command line applications. In the picture below, the general architecture is displayed.
-
-![general_architecture](docs/assets/images/architecture.png)
-
-The tools provided are:
-* [Viewer](https://jderobot.github.io/DetectionMetrics/functionality/viewer/): view the dataset images with the annotations.
-* [Detector](https://jderobot.github.io/DetectionMetrics/functionality/detector/): run a model over a dataset and get generate a new annotated dataset.
-* [Evaluator](https://jderobot.github.io/DetectionMetrics/functionality/evaluator/): evaluate the ground truth dataset with another one and get the comparison metrics.
-* [Deployer](https://jderobot.github.io/DetectionMetrics/functionality/deployer/): run a model over different inputs like a video or webcam and generate a new annotated dataset.
-* [Converter](https://jderobot.github.io/DetectionMetrics/functionality/converter/): convert a dataset into another dataset format.
-* [Command line application (CLI)](https://jderobot.github.io/DetectionMetrics/functionality/command_line_application/): access Detection Metrics toolset through command line
-* [Detection Metrics as ROS Node](https://jderobot.github.io/DetectionMetrics/functionality/ros_node/): use Detection Metrics as a ROS Node.
-+ [Labelling](https://jderobot.github.io/DetectionMetrics/resources/gsoc_19/): add or modify labels in the datasets in runtime when running Deployer.
-
-The idea is to offer a generic infrastructure to evaluate object detection models against a dataset and compute the common statistics:
-* mAP
-* mAR
-* Mean inference time.
-
-# What's supported in Detection Metrics.
-
-| Support | Detail                                                  |
-| ------ | ------------------------------------------------------------ |
-| Supported OS  | Multiplatform using Docker                 |
-| Supported datasets  | <ul><li>COCO</li><li>ImageNet</li><li>Pascal VOC</li><li>Jderobot recorder logs</li><li>Princeton RGB dataset</li><li>Spinello dataset</li><li>Open images dataset</li></ul> |                |
-| Supported frameworks   | <ul><li>TensorFlow</li><li>Keras</li><li>PyTorch</li><li>Yolo-OpenCV</li><li>Caffe</li><li>Background substraction</li></ul>  |
-| Supported inputs in Deployer   | <ul><li>WebCamera/USB Camera</li><li>Videos</li><li>Streams from ROS</li><li>Streams from ICE</li><li>JdeRobot Recorder Logs</li></ul> |
 
+# DetectionMetrics
```

Comment:
Since we're generating the new version, maybe we could move the docs/gh-pages to a new branch `gh-pages` to separate the docs from the actual application code. What do you think?

Reply:
I like the proposal. I'd try to merge it first and see if the current deployment works tho, because I have added plain HTML docs generated with Sphinx (`docs/py_docs`) and I'd like to check whether it's working or not.